### ISSUE TITLE
Authors not subscribed to posts > 30 days

### ISSUE THREAD

**Issue by terrizgit** [stcr/subscribe-to-comments-reloaded#293]
I've been having to manually remind authors to respond to comments - today I discovered they weren't being notified that there was a comment to reply to!

I looked at the subscriptions and I see that there are many for October, but few before that. My authors post about 5 times a week, so there should be many more before that. 

- I installed the plugin 2 years ago, so this isn't an issue of retroactive subscriptions.
- The plugin is up-to-date.
- "Subscribe Authors" is enabled.

I have "Autopurge requests" set to 30 days. Could this be why? Authors aren't sent double opt-in confirmation emails, so if this setting is looking at that, I'm guessing their subscriptions are being purged.

I just deactived and reactivated the plugin, to see if that fixes it. But even if it does - this seems like a bug.

Apologies if this is addressed elsewhere, I'm a GitHub noob and don't quite no my way around yet.

Thanks,
Terri Z

**Comment by Reedyseth** [comment]
Hello @terrizgit , Did you manage to solve the issue? Please feel free to reopen the issue if you still struggling to get this done.

Regards.!!

**Comment by terrizgit** [comment]
Hi,

The issue appears to have been fixed by my deactivation/reactivation of the plugin, I can't think of anything else I did. Authors are being subscribed as designed, since Oct 2016. And as some old posts get updated, it is subscribing authors retroactively. All good.

I did run into another snag though - I had 500 subscriptions (I have another plugin that creates multiple versions of an event, and it was subscribing the author to each one - 30 days before and after. I've just disabled comments for that plugin.) Is there a limit to the number of subscriptions that will be retained? I couldn't find an option that would set/change this.
